### PR TITLE
Add a new learning resource to community.md

### PR DESCRIPTION
**Pull request by Dedac**
Add a the video course Blazor First Look to the community learning resources.

Do I need to create an issue for this? 
Should I add something to indicate the course may require a paid account with LinkedIn?

Thanks for your time.

### CODE REVIEW

**Comment by danroth27** [comment]
Yes, I'm fine with add a Courses section.

I think it would be great actually to promote this LinkedIn course on the home page, kind of like how we sometimes have promotional banners at the top of https://asp.net. I chatted with @seanchor about this and she was going to propose some content for us to add as a temporary promotion.

**Comment by guardrex** [comment]
Ok, thanks. I'll add a *Courses* section to this PR.

I'm going to merge the other PR first. It's going to create a merge conflict over here, but I'll sort that out when the time comes. Stand-by for a bit while I get #228 squared away. I'll tend to this one next.

**Comment by guardrex** [comment]
@danroth27 This one is ready for a final :eye:.

File: community.md [hunk 1]
```diff
@@ -67,6 +67,10 @@ Below is a list of projects and resources from the Blazor community (links in ea
 * [Blazor Bites](https://codedaze.io/tag/blazor-bites/) &ndash; Posts on ASP.NET Core and Blazor subjects.
 * [Learn Blazor](https://learn-blazor.com/) &ndash; Community documentation on Blazor.
 
+## Courses
+
+* [Blazor First Look Video Course](https://www.linkedin.com/learning/blazor-first-look)
```
Comment (danroth27) [inline community.md]: @Dedac Do you want the text to include "LinkedIn Learning"? Something like: "LinkedIn Learning: Blazor First Look".

Comment (Dedac) [reply community.md]: @danroth27 I don't think it needs to, I will update the name a bit though

Comment (danroth27) [reply community.md]: I'm just imagining that we will probably end up with a bunch of courses here. How would you want the LinkedIn course to be distinguished from the others?

Comment (Dedac) [reply community.md]: You are right, I went with it :)

**Pull request merged**